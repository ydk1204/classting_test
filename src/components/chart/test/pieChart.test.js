import React from 'react';
import renderer from 'react-test-renderer';
import ScorePieChart from '../pieChart';

const answer = 7;
const wrong = 3;

describe('ScorePieChart', () => {
  it('renders', () => {
    const component = renderer.create(<ScorePieChart answer={answer} wrong={wrong} />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});