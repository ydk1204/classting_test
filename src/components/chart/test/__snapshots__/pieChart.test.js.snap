// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScorePieChart renders 1`] = `
<div
  style={
    Object {
      "height": 400,
      "width": 400,
    }
  }
>
  <div
    className="recharts-responsive-container"
    style={
      Object {
        "height": "100%",
        "maxHeight": undefined,
        "minHeight": undefined,
        "minWidth": undefined,
        "width": "100%",
      }
    }
  />
</div>
`;
